package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prodigy/navmesh",
-  "version": "1.0.1",
+  "version": "1.1.0-alpha-083c53c",
```

Comment:
Is this going to be the final version number?

Reply:
Nope, just JFrog things